release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3083,6 +3087,116 @@ Space precluded documenting all of the container images for this release in the
+
+[id="ocp-4-10-23-redfish-events"]
+==== Low-latency Redfish hardware event delivery (Technology Preview)
+
+{product-title} now provides a hardware event proxy that enables applications running on bare-metal clusters to respond quickly to Redfish hardware events, such as hardware changes and failures.
+
+The hardware event proxy supports a publish-subscribe service that allows relevant applications to consume hardware events detected by Redfish. The proxy must be running on hardware that supports Redfish v1.8 and higher. An Operator manages the lifecycle of the `hw-event-proxy` container.
+
+You can use a REST API to develop applications to consume and respond to events such as breaches of temperature thresholds, fan failure, disk loss, power outages, and memory failure. Reliable end-to-end messaging without persistent stores is based on the Advanced Message Queuing Protocol (AMQP). The latency of the messaging service is in the 10 millisecond range.
+
+[NOTE]
+====
+This feature is supported for single node OpenShift clusters only.
+====
+
+[id="ocp-4-10-23-ztp-ga"]
+==== Zero touch provisioning is generally available
+
+Use zero touch provisioning (ZTP) to provision distributed units at new edge sites in a disconnected environment. This feature was previously introduced as a Technology Preview feature in OpenShift Container Platform 4.9 and is now generally available and enabled by default in OpenShift Container Platform 4.11. For more information, see xref:../scalability_and_performance/ztp-deploying-disconnected.adoc[Deploying distributed units at scale in a disconnected environment].
+
+[id="ocp-4-10-23-enhancements-iod"]
+==== Indication of done for ZTP
+
+A new tool is available that simplifies the process of checking for a completed zero touch provisioning (ZTP) installation using the {rh-rhacm-first} static validator inform policy. It provides an Indication of done for ZTP installations by capturing the criteria for a completed installation and validating that it moves to a compliant state only when ZTP provisioning of the spoke cluster is complete.
+
+This policy can be used for deployments of single node clusters, three-node clusters, and standard clusters.
+To learn more about the validator inform policy, see xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#ztp-definition-of-done-for-ztp-installations_ztp-deploying-disconnected[Indication of done].
+
+[id="ocp-4-10-23-enhancements-ztp"]
+==== Enhancements to ZTP
+
+For {product-title} {product-version}, there are a number of updates that make it easier to configure the hub cluster and generate source CRs. New PTP and UEFI secure boot features for spoke clusters are also available. A summary of these features is below:
```

Comment:
Shouldn't use "below" in this way. Maybe:"The following is a summary of these features:"